### DISCUSSION TITLE
Two Mecha Sites in Main Folder and Subfolder, Not in New Subdomain

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#134] [Q&A]
Misalnya folder situs utama pakdul[dot]id saya install Mecha. Lalu saya buat folder di dalam situs utama tersebut, misalnya pakdul[dot]id/situsbaru dan menginstall situs Mecha yang lain. Apakah ini akan menjadikan mesin jadi rancu dan membingungkan?

Pasalnya, sama-sama menggunakan Mecha saya ingin membuat situs baru dari subfolder di bawah naungan situs utama ketimbang membuat subdomain.

*) Biar judulnya nggaya, pakai bahasa Inggris. Kalau salah tolong betulin ya, Mas. 😊

**Comment by taufik-nurrohman** [comment]
Bisa pak. Pakai fitur _addon domain_. Hostingan Saya juga sudah Saya atur seperti itu. Tiga situs web, di bagian _root_ Saya pakai untuk menampilkan situs web Mecha. Kemudian di situ Saya buat lagi folder `dte.web.id` dan `taufik-nurrohman.com` kemudian di dalamnya Saya pasang lagi Mecha. Untuk `taufik-nurrohman.com` malah masih memakai Mecha versi lawas. Dan itu sama sekali tidak masalah.

Nanti setiap folder biasanya akan ditangani oleh subdomain yang dibuat secara otomatis seperti `dte.web.id.mecha-cms.com`, kemudian kita bisa pakai referensi tersebut untuk membuat _addon domain_.

**Reply by ghost** [reply]
![IMG-20210712-WA0004](https://user-images.githubusercontent.com/53632644/125278112-80993800-e33c-11eb-91c5-1071dc6ac831.jpg)

Maksudnya begini, Mas:

Ada situs mainmecha.com menggunakan Mecha CMS. Di dalam situs mainmecha.com ada folder baru bernama "othermecha". Artinya, nanti kalau akses ke mainmecha.com/othermecha itu sebagai situs "Other Mecha" meski nunut situs utama "Main Mecha" karena situs "Other Mecha" menggunakan mesin Mecha CMS sendiri dengan isi yang berbeda dari situs "Main Mecha".

Yang saya pertanyakan ini akan membuat bingung mesin Mecha apa tidak?

Misalnya di dalam situs "Main Mecha" (di dalam folder page) saya buat folder lagi dengan nama yang sama: othermecha.

Gambarannya:
- mainmecha.com/othermecha (situs independen dalam subfolder)

- mainmecha.com/ punya page othermecha dan berisi tulisan berekstensi .page

Apakah mesin bisa membedakan mainmecha.com/othermecha sebagai laman beranda situs mandiri dengan mainmecha.com/othermecha/judul-artikel?

Sorry kalau saya njlimet menjelaskan dan membuat bingung Mas Taufik.

**Reply by taufik-nurrohman** [reply]
Nggak masalah pak. Yang ngatur semuanya mulai dari `.htaccess` di _root_. Kalau sebuah folder/file terdeteksi eksis, dia nggak bakalan pengaruh ke rute. Kalau folder/file terdeteksi nggak eksis, bakalan jatuh ke `RewriteRule` milik Apache dan diarahkan ke `index.php?_=/path/file/palsu`.

Nanti kueri `$_GET['_']` akan diproses oleh rute. Kalau folder/file eksis, nggak akan terjadi apa-apa. Contohnya seperti pas buka berkas dari folder `.\lot\asset` secara langsung. Nggak ada masalah kan?

**Comment by ghost** [comment]
Baik, Mas. Matur nuwun penjelasannya 😊
